source/material_model/rheology/visco_plastic.cc

```diff
@@ -299,7 +307,7 @@ namespace aspect
                   if (current_stress >= yield_stress)
                     {
                       viscosity_yield = drucker_prager_plasticity.compute_viscosity(current_cohesion,
-                                                                                    current_friction,
+                                                                                    output_parameters.current_friction_angles[j],
```

Comment:
Now you can keep current_friction here right? current_friction and output_parameters.current_friction_angles[j] are the same.